apps/web/i18n/routeNames.tsx

```diff
@@ -50,9 +52,17 @@ export type PathTypes =
   | 'LifeEventPage'
   | 'lifeEventPage'
   | 'adgerdir'
+  | 'vidspyrna-frontpage'
 
 export const routeNames = (locale: Locale = defaultLanguage) => {
   const makePath = (type?: PathTypes, subfix?: string) => {
+    let urlEnd
+    if (type === 'vidspyrna-frontpage') {
+      urlEnd = ''
+    } else {
+      urlEnd = subfix
+    }
```

Comment:
wouldn't this make more sense as a ternary? or is it planned to extend the if statements?
```suggestion
    let urlEnd = type === 'vidspyrna-frontpage' ? '' : subfix
```

Reply:
Optimally the link resolver should be able to resolve all links on web.
So It's likely this will be extended or replaced with a more "solid" approach at a later time